### PR TITLE
[13.0][MIG] hr_employee_service

### PR DESCRIPTION
**Pull request by Saran440**

### CODE REVIEW

File: hr_employee_service/__manifest__.py [hunk 1]
```diff
@@ -2,26 +2,16 @@
...
-        ],
-    },
-    'data': [
-        'views/hr_employee.xml',
-    ],
+    "name": "HR Employee Service",
+    "version": "13.0.1.0.0",
+    "category": "Human Resources",
+    "website": "https://github.com/OCA/hr",
+    "author": "Brainbean Apps, " "Odoo Community Association (OCA)",
```
Comment (alexey-pelykh) [inline hr_employee_service/__manifest__.py]: `" "` this looks not great?

@pedrobaeza can we do something about these cases with autoformatter?

File: hr_employee_service/models/hr_employee.py [hunk 2]
```diff
@@ -1,78 +1,75 @@
...
-        string='Hire Date',
-        groups='hr.group_hr_user',
-        track_visibility='onchange',
+        string="Hire Date",
+        groups="hr.group_hr_user",
+        track_visibility="onchange",
         help=(
-            'Hire date is normally the date an employee completes new hire'
-            ' paperwork'
+            "Hire date is normally the date an employee completes new hire" " paperwork"
```
Comment (alexey-pelykh) [inline hr_employee_service/models/hr_employee.py]: `" "` as well

**Review by alexey-pelykh — APPROVED** [review]
Good overall, one generic question

File: hr_employee_service/__manifest__.py (hunk 1 above)
Comment (pedrobaeza) [reply hr_employee_service/__manifest__.py]: No, except manual change

Comment (alexey-pelykh) [reply hr_employee_service/__manifest__.py]: Do OCA has any opinion on that cosmetic issue?

**Comment by alexey-pelykh** [comment]
@Saran440 please rebase

**Comment by alexey-pelykh** [comment]
Looks quite merge-able to me

**Review by lfreeke — APPROVED** [review]
Functional test :+1:

**Pull request merged**